content/en/feature_flags/client/browser.md

```diff
@@ -0,0 +1,216 @@
+---
+title: Browser Feature Flags
+description: Set up Datadog Feature Flags for browser applications.
+aliases:
+  - /feature_flags/setup/browser/
+further_reading:
+- link: "/feature_flags/client/"
+  tag: "Documentation"
+  text: "Client-Side Feature Flags"
+- link: "/real_user_monitoring/browser/"
+  tag: "Documentation"
+  text: "Browser Monitoring"
+---
+
+{{< callout url="http://datadoghq.com/product-preview/feature-flags/" >}}
+Feature Flags are in Preview. Complete the form to request access.
+{{< /callout >}}
+
+## Overview
+
+This page describes how to instrument your browser application with the Datadog Feature Flags SDK. Datadog feature flags provide a unified way to remotely control feature availability in your app, experiment safely, and deliver new experiences with confidence.
+
+The Datadog Browser Feature Flags SDK is built on [OpenFeature][1], providing a standardized API for feature flag evaluation. This guide explains how to install and configure the SDK, set up evaluation context, and evaluate flags in your application.
+
+## Installation
+
+Install the Datadog OpenFeature Browser provider and the OpenFeature Web SDK:
+
+{{< tabs >}}
+{{% tab "npm" %}}
+{{< code-block lang="bash" >}}
+npm install @datadog/openfeature-browser @openfeature/web-sdk
+{{< /code-block >}}
+{{% /tab %}}
+
+{{% tab "yarn" %}}
+{{< code-block lang="bash" >}}
+yarn add @datadog/openfeature-browser @openfeature/web-sdk
```

Comment:
`yarn` requires manually installing the`@openfeature.core` peer dep as well (npm does not)